Algebra.java

```diff
@@ -25,43 +25,187 @@ public static void main(String args[]) {
 
 	// Returns x1 + x2
 	public static int plus(int x1, int x2) {
-		// Replace the following statement with your code
-		return 0;
+			if (x1 < 0 && x2 < 0)
+		{
+			for( int i = x2; i <  0; i++)
+		{
+			x1 -= 1;
```

Comment:
supposed to use x1--.. 
Since logically it is the same I will not deduct it, 
If you are not sure contact use